software/apps/audio_module/event_detector/dynArray.c

```diff
@@ -0,0 +1,64 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "dynArray.h"
+#include "_kiss_fft_guts.h"
+
+void initArray(Array *a, size_t initialSize) {
+    a->maxSize = initialSize;
+    a->used = 0;
+
+    a->size = a->maxSize;
+    printf("Init array size = %zu\n",a->size);
```

Comment:
Thank you so much for letting me know about DEBUG_PRINTF. This will be fixed in the next pull request.